CHANGELOG.md

```diff
@@ -41,6 +41,7 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Archived local observations not visible on cancer variantS page
 - Empty Population Frequency column in the Cancer SV Variants view
 - Capital letters in ClinVar events description shown on case page
+- Optimised indexes to address slow queries
```

Comment:
Don't forget to fix the changelog by moving this to the new release before merging!